src/lib.rs

```diff
@@ -21,13 +21,22 @@ use log;
 
 pub const STORAGES_FOLDER: &str = ".ark";
 
-// allowed to be lost (cache)
-pub const INDEX_PATH: &str = "index";
-pub const PREVIEWS_PATH: &str = "previews";
-
 // must not be lost (user data)
 pub const METADATA_PATH: &str = "meta";
 pub const TAGS_PATH: &str = "tags";
```

Comment:
These two must be replaced by `METADATA_STORAGE_FOLDER` and `TAG_STORAGE_FILE` correspondingly.